index.bs

```diff
@@ -933,6 +935,35 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+### Platform Authenticator Availability - PublicKeyCredential's `isPlatformAuthenticatorAvailable()` method ### {#isPlatformAuthenticatorAvailable}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorAvailable">
+
+[=[RPS]=] use this method to determine whether they can create a new credential using a [=platform authenticator=].
+Upon invocation, the [=client=] employs a platform-specific procedure to discover available [=platform authenticators=].
+If successful, the [=client=] then assesses whether the user is willing to create a credential using one of the available [=platform authenticators=].
+This assessment may include various factors, such as:
+    - Whether the user is running in private or incognito mode.
+    - Whether the user has configured the [=client=] to not create such credentials.
+    - Whether the user has previously expressed an unwillingness to create a new credential for this [=[RP]=],
+        either through configuration or by declining a user interface prompt.
+    - The user's explicitly stated intentions, determined through user interaction.
+If this assessment is affirmative, the promise is resolved with the value of <i>True</i>.
+Otherwise, the promise is resolved with the value of <i>False</i>.
```

Comment:
I don't think that use cases in which someone would use a user-verifying platform authenticator overlap with use cases in which someone would use a non-platform authenticator.

To repeat @kpaulh 's question: Do we no longer have the goal of hiding the existence-vs-non-existence of authenticators from RPs unless the user approves? If we don't care, then I think we should take a different approach altogether - a GetAuthenticatorInfo() call or some such thing. But I think we *should* care, and that means that the two different error conditions (authenticator-doesn't-exist vs authenticator-exists-but-user-declines-use) should be indistinguishable.